spec/javascript/containers/AirlineShowContainer.js

```diff
@@ -0,0 +1,48 @@
+import AirlineShowContainer from '../../../app/javascript/react/container/AirlineShowContainer';
+import AirlineShowTile  from '../../../app/javascript/react/components/AirlineShowTile'
+import { mount } from 'enzyme';
+import jasmineEnzyme from 'jasmine-enzyme';
+import React from 'react';
+import fetchMock from 'fetch-mock'
+
+describe('AirlineShowContainer', () => {
+ let wrapper;
+const props = {
+  id: 1,
+ airline: "test",
+destinations: "test",
```

Comment:
Please change these to different values -- otherwise, you might be trying to test whether the description is displaying and get a false positive because it's seeing the airline name

Reply:
noted -- thank you.